### PR TITLE
STCOM-501: Add basic accordion component

### PR DESCRIPTION
**Pull request by Yurii-Danylenko**
https://issues.folio.org/browse/STCOM-501

## Purpose
Create basic accordion for notes with MCL table,  "Add" and New"  buttons, which will be used to manage work with notes inside the UI module where notes used. It's the beginning part of the whole component. Next step is to add embed modal with possibility to search and attach notes.

## Approach
- Create Accordion
- Create NotesList component
- Add tests
- Add translations
- Add storybook

## TODO
- Add modal component for attaching notes

### CODE REVIEW

File: lib/NotesAccordion/readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# NotesAccordion
+
+Accordion enhanced to work with Notes
+
+## Basic Usage
+
+```js
+  <NoteAccordion
```
Comment (zburke) [inline lib/NotesAccordion/readme.md]:
```
<NotesAccordion...
     ^
     ^
```

Comment (Yurii-Danylenko) [reply lib/NotesAccordion/readme.md]: Ups, thank you @zburke!

**Comment by JohnC-80** [comment]
This seems more specific to Notes functionality than what we usually like to keep in stripes-components. We'd like for new components to be functional at the smallest level rather than larger structures/molecules. Are there still plans for a smart-component? Despite not being connected to data, it seems better to have this live in a notes module/npm package/smart-component than here.

**Comment by Yurii-Danylenko** [comment]
From the conversation with John, it was decided to move notes accordion to smart components

**Pull request closed without merging**